closure.conf

```diff
@@ -11,6 +11,7 @@
 --rewrite_polyfills=false
 --module_resolution=node
 
+generic_externs.js
```

Comment:
this is a pretty gross workaround, because everyone would have to copy such a file into their project. Let's wait a couple days to see if we get it added to the upstream externs?